tests/log_manager_test.cc

```diff
@@ -76,11 +76,11 @@ TEST(LogManagerTest, TestLeaderPushLog) {
             leaderFlag = i;
         }
 
-        ASSERT_EQ(nodes[i]->CurrLogIndex(), -1);
+        ASSERT_EQ(nodes[i]->CurrLogIndex(), 0);
```

Comment:
Why change -1 to 0 as the currentIndex is initialized to -1 in LeaderLogManager.

Reply:
leader has a no_op_request log